light_node/src/main.rs

```diff
@@ -518,85 +568,81 @@ fn main() {
 
     // create common RocksDB block cache to be shared among column families
     // IMPORTANT: Cache object must live at least as long as DB (returned by open_kv)
-    let cache = Cache::new_lru_cache(128 * 1024 * 1024).unwrap(); // 128 MB
-
-    let schemas = vec![
-        block_storage::BlockPrimaryIndex::descriptor(&cache),
-        block_storage::BlockByLevelIndex::descriptor(&cache),
-        block_storage::BlockByContextHashIndex::descriptor(&cache),
-        BlockMetaStorage::descriptor(&cache),
-        OperationsStorage::descriptor(&cache),
-        OperationsMetaStorage::descriptor(&cache),
-        context_action_storage::ContextActionByBlockHashIndex::descriptor(&cache),
-        context_action_storage::ContextActionByContractIndex::descriptor(&cache),
-        context_action_storage::ContextActionByTypeIndex::descriptor(&cache),
-        ContextActionStorage::descriptor(&cache),
-        MerkleStorage::descriptor(&cache),
-        SystemStorage::descriptor(&cache),
-        Sequences::descriptor(&cache),
-        MempoolStorage::descriptor(&cache),
-        ChainMetaStorage::descriptor(&cache),
-        PredecessorStorage::descriptor(&cache),
+    let cache = [
```

Comment:
@mat13mn 
if we dont need cache later, I would move this cache initialization inside `initialize_databases`

Reply:
that was also my idea but i found this note in the code :/
```
    // create common RocksDB block cache to be shared among column families
    // IMPORTANT: Cache object must live at least as long as DB (returned by open_kv)
```

https://github.com/simplestaking/tezedge/blob/master/light_node/src/main.rs#L513